### PR TITLE
Favor language tour over article for mixins info

### PR DESCRIPTION
**Pull request by kwalrath**
Staged:
https://kw-www-dartlang-1.firebaseapp.com/articles/language/mixins
https://kw-www-dartlang-1.firebaseapp.com/articles/language

### CODE REVIEW

File: src/_articles/language/mixins.md [hunk 1]
```diff
@@ -2,18 +2,27 @@
...
+obsolete: true
 ---
 
 _Written by Gilad Bracha <br>
 December 2012 (updated June 2018)_
 
-Looking for an introduction to mixins?
-See the Language Tour section [Adding features to a class: mixins][].
+This article provides a brief theoretical presentation of
+the evolution of mixins in Dart.
```
Comment (vsmenon) [inline src/_articles/language/mixins.md]: Do you want to keep the old link to the lang tour?  E.g., something like: "For a guide on how to use mixins today, please see ... instead" or similar?

Comment (kwalrath) [reply src/_articles/language/mixins.md]: I moved that link to a note, hoping it'd be more noticeable. Did you look at the staged page?

https://kw-www-dartlang-1.firebaseapp.com/articles/language/mixins

Comment (vsmenon) [reply src/_articles/language/mixins.md]: got it - thanks!

**Comment by vsmenon** [comment]
lgtm!

**Pull request merged**